openapi.yaml

```diff
@@ -2815,44 +2682,6 @@ components:
             - $ref: '#/components/schemas/apparent-power-value'
             - $ref: '#/components/schemas/current-value'
             - $ref: '#/components/schemas/actual-power-and-factor-value'
-            
-    rating-input:
-      description: General type for a rating input
-      oneOf:
-      - $ref: '#/components/schemas/aar-input'
-      - $ref: '#/components/schemas/dlr-input'
-
-    aar-input:
-      description: |
-        Ambient-adjusted rating, described as a minimum requirement for FERC 881.  Includes
-        a temperature, and an optional forecast source, such as a weather station or 
-        forecasted weather coordinate.  
-      type: object
-      additionalProperties: false
-      properties:
-        weather-source:
-          $ref: '#/components/schemas/generic-identifier'
-        degrees-fahrenheit:
```

Comment:
I'm trying to find out if it is really a requirement, but it has been suggested that providing the temp that was used by the RC in a lookup table would be necessary to support TOP compliance. We can address this later.